app/views/shared/_footer.haml

```diff
@@ -1,12 +1,20 @@
 .footer
-  - if !current_project.nil? && !current_project.program.email.blank?
-    .help 
-      For assistance with current competition,
+  .help
+    - if (controller_name == 'projects' && action_name != 'index') || controller_name == 'submissions'
+      For assistance with this competition,
       %b
         %i
-          = link_to(current_project.project_title, show_competition_url(current_project.program.program_name, current_project.project_name)) + "," 
+          = link_to(current_project.project_title, show_competition_url(current_project.program.program_name, current_project.project_name)) + ","
           e-mail
       %a{href: "mailto:#{current_project.program.email}?subject=[NITRO Competitions] Assistance with #{current_project.project_title}"}
         = current_project.program.email + "."
+    - elsif current_page? controller: 'sponsors', action: 'show'
+      %p
+        Questions? Contact
+        = precede ' ' do
+          %a{href: "mailto:#{@sponsor.email}?subject=[NITRO Competitions] Contact Question"}>#{@sponsor.program_title}
```

Comment:
What if sponsor email is null? We shouldn't show this. 

Same for sponsor program_title.